core/templates/domain/admin/admin-backend-api.service.ts

```diff
@@ -304,23 +319,13 @@ export class AdminBackendApiService {
   }
 
   // Admin Misc Tab Services.
-  async clearSearchIndexAsync(): Promise<void> {
+  async clearSearchIndexAsync(): Promise<PopulateStatsResult> {
```

Comment:
Why change in type here from `<void>` to `<PopulateStatsResult>`.
(Doesn't make any sense)

Reply:
@DubeySandeep thoughts?

Reply:
Try to log the data returned here by the backend. This interface is apt here.

Reply:
I'm getting empty dicts.
![Screenshot from 2021-06-27 02-41-13](https://user-images.githubusercontent.com/54100702/123525834-4a7b7780-d6f1-11eb-93ff-28a7e45328ae.png)

Reply:
Check every handler. If all of them are returning empty dicts. Revert the data types back to void.

Reply:
can you fix it?

Reply:
Ok, got it. A handler /populateexplorationstats was returning data. So, i added this interface but later during the review, sean asked to remove the "the populate exploration stats" feature. So, i think this interface is not required now. Please, revert it back to void. But check all responses once.

Reply:
> can you fix it?

If you working on it at the moment, please go ahead. I might be able to work on this tomorrow evening.

Reply:
In that case i ill fix it

Reply:
This is fixed through https://github.com/oppia/oppia/pull/13230